OpenSim/Simulation/Model/Appearance.h

```diff
@@ -59,8 +59,8 @@ class OSIMSIMULATION_API SurfaceProperties : public Object {
     //==========================================================================
     OpenSim_DECLARE_PROPERTY(representation, int,
      "The representation (1:Points, 2:Wire, 3:Shaded) used to display the object.");
-    OpenSim_DECLARE_OPTIONAL_PROPERTY(texture_file, std::string,
-        "Name of file containing texture.");
+    OpenSim_DECLARE_OPTIONAL_PROPERTY(texture, std::string,
+        "Name of texture e.g. metal, bone. This is a hint to the GUI/Visualizer, implementation is Visualization dependent.");
```

Comment:
@aseth1 The change here is moving from a specific file for texture (which was available in old models/format) to a name based texture that can be reused and that's why the comment was updated. @tkuchida supported textures will depend on the visualizer, e.g. API visualizer supports none while I foresee GUI supporting "bone", "sand", "metal", "wood" etc. None of these is implemented yet pending this change (and also texture coordinates assigned to geometry) but we should open an issue to make sure that this important documentation doesn't fall through the cracks.